optaplanner-benchmarks/pom.xml

```
@@ -44,6 +49,34 @@
         <artifactId>jmh-generator-annprocess</artifactId>
         <version>${version.jmh}</version>
       </dependency>
+      <!-- Optaplanner -->
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-bom</artifactId>
+        <version>${version.org.optaplanner}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-build-parent</artifactId>
+        <version>${version.org.optaplanner}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.0</version>
+        <configuration>
+          <release>${maven.compiler.jdk.version}</release>
```

```suggestion
          <source>1.8</source>
          <target>1.8</target>
```

This will make the project build with Java 8.

---

Ah, sure 

> I have one more request. Maven prints the following warning at the beginning of the build:
> 
> ```
> [WARNING] Some problems were encountered while building the effective model for org.kie:optaplanner-perf-benchmark:jar:1.0-SNAPSHOT
> [WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-shade-plugin is missing. @ line 54, column 15
> ```
> 
> Could you resolve that by defining the Shade plugin version?

e624da07

---

Yes, thanks, when I notice the falling CI I immidiatly did that, but then I was puzzled that we actually want jdk 11 here, but seems like the code does not contain any jdk 11 features so I can just use jdk 8 and use java 11 to run benchmarks.

---

Yup. The benchmarks project compiles fine with Java 8 (although OptaPlanner itself requires Java 11).

---

Resolved.

---

e624da07